vault/overlays/nerc-ocp-infra/auto-unseal/cronjob.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: auto-unseal-vault
+spec:
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 1
+  schedule: "0 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        metadata:
+          name: auto-unseal-vault
+        spec:
+          serviceAccountName: auto-unseal-job
+          restartPolicy: Never
+          containers:
+            - name: auto-unseal-vault
+              image: quay.io/dystewar/vault-unseal:latest
+              imagePullPolicy: IfNotPresent
+              command: ["bash"]
+              args:
+                - /scripts/check-vault-seal.sh
+              volumeMounts:
+                - mountPath: scripts
+                  name: scripts
+          volumes:
```

Comment:
If we're using a custom image already, why are we mounting this script via a configmap rather than just including it in the image?

Reply:
In the meantime I included the script via cm for clarity, but it certainly makes sense to bake it into the image. Not to mention once the image moves and the sources are added to the repo it'll still be clear anyway. I'll make this change